tests/response/test_response_headers.py

```diff
@@ -72,3 +73,27 @@ def my_handler(data: Dict[str, str]) -> Dict[str, str]:
         response = client.post("/test", json={"hello": "world"})
         assert response.status_code == HTTP_201_CREATED
         assert response.headers.get("test-header") == "test value"
+
+
```

Comment:
This should probably include tests for overwrite behaviour between `response_headers=` and `cache_control=`. It's mentioned in the docs, so accidentally changing the overwrite cascade in some way could break things if people rely on the specific order.

Reply:
Sure, the test was added here: https://github.com/starlite-api/starlite/pull/601/commits/56a8814ae9e7b1a30bc34b00d4236cbc82734fad